lib/internal/Magento/Framework/App/Http.php

```diff
@@ -150,7 +150,7 @@ public function catchException(Bootstrap $bootstrap, \Exception $exception)
      */
     private function handleDeveloperMode(Bootstrap $bootstrap, \Exception $exception)
     {
-        if ($bootstrap->isDeveloperMode()) {
+        if (!$bootstrap->isDeveloperMode()) {
```

Comment:
@cspruiell: Most likely a left-over from an inversion (exit early), I'll correct this and update the PR.